### PR TITLE
[ADD] openacademy: beginning of the module implementation

### PR DESCRIPTION
**Pull request by luasem**
Began with the implementation of the openacademy module by using odoo-bin scaffold command, and adding only the course model definition.

### CODE REVIEW

**Comment by luasem** [comment]
@lescobarvx

File: openacademy/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```
Comment (luiseevaquer) [inline openacademy/__init__.py]: This line with code utf-8 is not needed anymore

File: openacademy/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Open Academy",
+
+    'summary': """
+        Short (1 phrase/line) summary of the module's purpose, used as
+        subtitle on modules listing or apps.openerp.com""",
+
+    'description': """
```
Comment (luiseevaquer) [inline openacademy/__manifest__.py]: the key 'description' is not necessary, only summary is required

File: openacademy/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    """,
+
+    'author': "My Company",
+    'website': "http://www.yourcompany.com",
+
+    # Categories can be used to filter modules in modules listing
+    # Check https://github.com/odoo/odoo/blob/15.0/odoo/addons/base/data/ir_module_category_data.xml
+    # for the full list
+    'category': 'Uncategorized',
+    'version': '0.1',
```
Comment (luiseevaquer) [inline openacademy/__manifest__.py]: version format is xx.xx.xx.xx.xx, for example: 15.0.1.0.1

File: openacademy/controllers/__init__.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (luiseevaquer) [inline openacademy/controllers/__init__.py]: Same as __manifest__ file

File: openacademy/controllers/controllers.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```
Comment (luiseevaquer) [inline openacademy/controllers/controllers.py]: Same

File: openacademy/demo/demo.xml [hunk 6]
```diff
@@ -0,0 +1,30 @@
+<odoo>
+    <data>
```
Comment (luiseevaquer) [inline openacademy/demo/demo.xml]: tag data is not longer needed

File: openacademy/models/course.py [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```
Comment (luiseevaquer) [inline openacademy/models/course.py]: Same

File: openacademy/models/models.py [hunk 8]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```
Comment (luiseevaquer) [inline openacademy/models/models.py]: Same

File: openacademy/views/templates.xml [hunk 9]
```diff
@@ -0,0 +1,24 @@
+<odoo>
+    <data>
```
Comment (luiseevaquer) [inline openacademy/views/templates.xml]: Same as demo.xml file

File: openacademy/views/views.xml [hunk 10]
```diff
@@ -0,0 +1,60 @@
+<odoo>
+  <data>
```
Comment (luiseevaquer) [inline openacademy/views/views.xml]: Same

**Comment by luasem** [comment]
Finished fixes and implementing the module's structure according to [this guide](https://www.odoo.com/documentation/15.0/developer/howtos/backend.html#build-an-odoo-module); next step is to begin work on the [model's views](https://www.odoo.com/documentation/15.0/developer/howtos/backend.html#basic-views).
@lescobarvx

File: .gitignore [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+.isort.cfg
+.pre-commit-config-autofix.yaml
+.pre-commit-config-optional.yaml
+.pre-commit-config.yaml
+.prettierrc.yml
+.pylintrc
+.pylintrc-optional
+**/__pycache__/*
+.python-version
+pyproject.toml
```
Comment (luiseevaquer) [inline .gitignore]: @luasem 

Please move this file to your local setting

**Comment by luiseevaquer** [comment]
@luasem 

LGTM  :+1:

**Pull request merged**